packages/flutter/lib/src/material/typography.dart

```diff
@@ -77,6 +78,32 @@ class TextTheme {
       caption  = const TextStyle(fontFamily: 'Roboto', inherit: false, fontSize:  12.0, fontWeight: FontWeight.w400, color: Colors.white70, textBaseline: TextBaseline.alphabetic),
       button   = const TextStyle(fontFamily: 'Roboto', inherit: false, fontSize:  14.0, fontWeight: FontWeight.w500, color: Colors.white,   textBaseline: TextBaseline.alphabetic);
 
```

Comment:
rename _black and _white as well.

Maybe blackMountainView and blackCupertino etc.